apps/admin-gui/src/app/admin/pages/admin-page/admin-visualizer/user-destination-relationship/user-destination-graph/user-destination-graph.component.ts

```diff
@@ -317,31 +360,31 @@ export class UserDestinationGraphComponent implements OnInit {
     return false;
   }
 
-  findConnectedGroups(groups1: Group[], groups2: Group[]): Group[] {
+  private findConnectedGroups(groups1: Group[], groups2: Group[]): Group[] {
     const result: Group[] = [];
-    for (let i = 0; i < groups1.length; i++) {
-      for (let j = 0; j < groups2.length; j++) {
-        if (groups1[i].id === groups2[j].id) {
-          result.push(groups1[i]);
+    for (const item of groups1) {
+      for (const item1 of groups2) {
```

Comment:
I know this whole file has this unfortunately called variables, but `item` and `item1`?